.sail/Dockerfile

```
@@ -1,14 +1,17 @@
 FROM codercom/ubuntu-dev-go:latest
 
```

Change the shell to bash so we can source the `nvm.sh` later.
```suggestion
SHELL ["/bin/bash", "-c"]
```